tmp/wrapper_generated.py

```diff
@@ -0,0 +1,284 @@
+    {
+      static_cast<coord_t>(M),
+      static_cast<coord_t>(N),
+      static_cast<coord_t>(K),
+      static_cast<coord_t>(B)
+    }, // ProblemShape problem_shape
+    {
+      (cutlass::half_t*)(X),  // ElementA const* ptr_A
+      {
+        32L /* stride_x0 */,
+        cute::Int<1>{} /* stride_x1 */,
+        0 /* batch_stride_x */
+      },  // StrideA dA
+      (cutlass::half_t*)(W),  // ElementB const* ptr_B
+      {
+        cute::Int<1>{} /* stride_w1 */,
+        256L /* stride_w0 */,
+        0 /* batch_stride_w */
+      },  // StrideB dB
+    },  // MainloopArguments mainloop
+
+    // see https://github.com/NVIDIA/cutlass/blob/e0aaa3c3b38db9a89c31f04fef91e92123ad5e2e/include/cutlass/epilogue/collective/sm90_epilogue_tma_warpspecialized.hpp#L184
+    {
+      {{}, { static_cast<ElementAcc>(3.0) }},  // thread, typename FusionCallbacks::Arguments ( EVT Arguments )
+      nullptr,  // ElementC const* ptr_C
+      {
+        cute::Int<1>{} /* stride_bias0 */,
+        cute::Int<1>{} /* stride_bias1 */,
+        0 /* batch_stride_bias */
+      },  // StrideC dC
+      (cutlass::half_t*)(Y),  // ElementD const* ptr_D
+      {
+        256L /* stride_y0 */,
+        cute::Int<1>{} /* stride_y1 */,
+        0 /* batch_stride_y */
+      },  // StrideD dD
+    },  // EpilogueArguments epilogue
+  };
+  cutlass3x_sm90_tensorop_h64x128x16gemm_f16_f16_f16_void_f16_128x128x64_2x1x1_0_ttn_align8_warpspecialized_cooperative_epi_tma_device_type gemm_op;
+  if (workspace_size) {
+    *workspace_size = gemm_op.get_workspace_size(arguments);
+    return 0;
+  }
+  {
+    auto status = gemm_op.can_implement(arguments);
+    CUTLASS_CHECK(status);
+  }
+  {
+    auto status = gemm_op.initialize(arguments, workspace, stream);
+    CUTLASS_CHECK(status);
+  }
+  {
+    auto status = gemm_op(stream);
+    CUTLASS_CHECK(status);
+  }
+  }
+  catch (std::exception& e) {
+    std::cerr << "Runtime error: " << e.what() << std::endl;
+    return -1;
+  }
+  catch (...) {
+    return -1;
+  }
+  return 0;
+}
+}
+''', 'so')
+
+import triton
+import triton.language as tl
+from torch._inductor.triton_heuristics import grid, start_graph, end_graph
+from torch._C import _cuda_getCurrentRawStream as get_cuda_stream
+
+
+async_compile.wait(globals())
+del async_compile
+
+def call(args):
+    arg0_1, arg1_1 = args
+    args.clear()
+    assert_size_stride(arg0_1, (256, 32), (32, 1))
+    assert_size_stride(arg1_1, (32, 256), (256, 1))
+    with torch.cuda._DeviceGuard(0):
+        torch.cuda.set_device(0) # no-op to ensure context
+        buf1 = empty_strided((256, 256), (256, 1), device='cuda', dtype=torch.float16)
+        buf4 = empty_strided((256, 256), (256, 1), device='cuda', dtype=torch.float16)
+        # Source Nodes: [mul], Original ATen: [aten.mul]
+        stream0 = get_cuda_stream(0)
+        cuda_fused_mul_0.cuda_fused_mul_0(c_void_p(arg0_1.data_ptr()), c_void_p(arg1_1.data_ptr()), c_void_p(buf4.data_ptr()), None, None, c_void_p(stream0))
```

Comment:
Call into the fused kernel, writing the results in the output buffer of the multiplication